docs/use_a_template.md

```diff
@@ -21,11 +21,15 @@ If you don't want to download the manifest file locally, you can point to its re
 If you are using InfluxDB Cloud platform you will need provide the URL of your InfluxDB zone using the `--host` flag. For example `https://us-west-2-1.aws.cloud2.influxdata.com`. 
 
```

Comment:
You could add the following link to the cloud URLs doc: https://v2.docs.influxdata.com/v2.0/cloud/urls/